doc/source/user_guide/tui_commands.rst

```diff
@@ -1,38 +1,41 @@
 .. _ref_user_guide_tui_commands:
 
-Using TUI Commands
+Using TUI commands
 ==================
 
-TUI commands refers to a programming interface that mirrors the Fluent text user
-interface (TUI). There is a TUI command hierarchy defined for each of meshing and solution mode -
-which hierarchy is active depends on the current Fluent mode. The guidance here is 
-applicable to either mode.
+TUI commands refer to a programming interface that mirrors the Fluent TUI (text user
+interface). There is a TUI command hierarchy defined for each of the two modes, meshing
+and solution. The hierarchy that is active depends on the current Fluent mode. The guidance
+in this topic applies to both modes.
 
-The PyFluent TUI commands represent a means to automate workflows comprehensively: everything
+The PyFluent TUI commands allow you to automate workflows. Everything
 that's in the Fluent TUI (which itself is a comprehensive automation interface)
-is exposed in PyFluent. The PyFluent TUI commands are Pythonic versions of those used
-in the Fluent console.
+is exposed in PyFluent. The PyFluent TUI commands are Pythonic versions of the
+commands that are used in the Fluent console.
 
 The PyFluent TUI commands do not support TUI features such as aliases or
-command abbreviation. As an alternative, using these PyFluent commands in an interactive
-session is easier if you install a tool such as
-`pyreadline3 <https://github.com/pyreadline3/pyreadline3>`_ which provides
-both command line completion and history. You can also use Python built-in functions,
+command abbreviation. To make using PyFluent commands in an interactive
+session easier, you can install a tool such as
+`pyreadline3 <https://github.com/pyreadline3/pyreadline3>`_, which provides
+both command line completion and history. To inspect any PyFluent TUI object further,
+you can also use the Python built-in
 `help <https://docs.python.org/3/library/functions.html#help>`_ and 
-`dir <https://docs.python.org/3/library/functions.html#dir>`_ on any PyFluent TUI object to inspect it further.
+`dir <https://docs.python.org/3/library/functions.html#dir>`_ functions.
 
-The arguments to a TUI command are just those that would be passed in direct interaction with the
-Fluent Console, but in a Pythonic style. Let's look in more detail at how the Python usage mirrors 
-existing TUI usage, because the most productive way to write Python commands is with reference to
-existing TUI commands. For instance, in solution mode:
+The arguments to a TUI command are just those that would be passed in direct interaction in the
+Fluent console, but in a Pythonic style. The most productive way to write Python commands
+is with reference to existing TUI commands. The following examples show how the Python usage
+mirrors the existing TUI usage.
+
+Assume in the solution mode, you typed the following in the Fluent console to set
+velocity inlet properties:
 
 .. code:: lisp
```

Comment:
I guess we can write `scheme` here instead of `lisp`:

https://pygments.org/docs/lexers/#pygments.lexers.lisp.SchemeLexer